bundles/org.openhab.ui/web/src/assets/definitions/widgets/home/index.js

```diff
@@ -21,6 +21,32 @@ export const OhHomePageDefinition = () => new WidgetDefinition('oh-home-page', '
       ]).m()
   ])
 
+const EquipmentListParameterGroup = () => pg('equipmentList', 'Equipment List', 'General settings for equipment lists in this tab')
+
+const EquipmentListParameters = () => [
+  pt('equipmentNesting', 'Display sub-equipment levels')
+    .o([
+      { value: 'nested', label: 'Nested pages for sub-equipment (default)' },
+      { value: 'accordion', label: 'Equipment items grouped as accordion cards' }
```

Comment:
Wouldn't it be better to display the 3 parameters below only when `equipmentNesting === 'accordion'` witb `v()` (if I understand correctly they are only applicable in that case)?